pkgs/by-name/sc/scalene/package.nix

```diff
@@ -0,0 +1,48 @@
+{ lib
```

Comment:
This file appears to be a virtual copy of `python-modules/scalene/default.nix`, so consider deleting it and in `top-level/all-packages.nix` and then applying `toPythonApplication` to it in `top-level/all-packages.nix`.

Reply:
It is a copy, and I didn't know I could do that! Thank you.